Source/WebCore/platform/TextRecognitionResult.h

```diff
@@ -110,10 +112,8 @@ struct TextRecognitionResult {
     Vector<TextRecognitionBlockData> blocks;
 
 #if ENABLE(IMAGE_ANALYSIS_ENHANCEMENTS)
-    RetainPtr<NSData> imageAnalysisData;
-
-    WEBCORE_EXPORT static RetainPtr<NSData> encodeVKCImageAnalysis(RetainPtr<VKCImageAnalysis>);
-    WEBCORE_EXPORT static RetainPtr<VKCImageAnalysis> decodeVKCImageAnalysis(RetainPtr<NSData>);
+    std::optional<WebCore::AttributedString> imageAnalysisData;
+    WEBCORE_EXPORT static std::optional<WebCore::AttributedString> extractAttributedString(RetainPtr<VKCImageAnalysis>);
```

Comment:
Let's go with one of these two:

```suggestion
    WEBCORE_EXPORT static std::optional<WebCore::AttributedString> extractAttributedString(VKCImageAnalysis *);
```

```suggestion
    WEBCORE_EXPORT static std::optional<WebCore::AttributedString> extractAttributedString(RetainPtr<VKCImageAnalysis>&&);
```

(We generally avoid passing around `RetainPtr` as arguments by copy)